packages/@react-aria/i18n/src/index.ts

```diff
@@ -18,6 +18,7 @@ export {useDateFormatter} from './useDateFormatter';
 export {useNumberFormatter} from './useNumberFormatter';
 export {useCollator} from './useCollator';
 export {useFilter} from './useFilter';
+export {isRTL} from './utils';
```

Comment:
Seems fine, however, i was looking at their example, couldn't they just move the i18n provider outside of the `html` element then consume `useLocale` to get the direction? then there's no worry about it not updating accidentally or something like that?